CHANGELOG.md

```diff
@@ -1,22 +1,18 @@
 # Changelog for ROCgdb
 
 Full documentation for ROCgdb is available at
-[docs.amd.com](https://docs.amd.com/bundle/rocgdb)
+[rocm.docs.amd.com/rocgdb](https://rocm.docs.amd.com/projects/ROCgdb/en/latest/index.html)
 
 ## ROCgdb-16
```

Comment:
@lancesix This needs to be replaced with "rocgdb version number for ROCm 6.3". Please let me know which headings are applicable to be listed under 6.3 along with the rocgdb version number.